examples/vision/text_image_denoiser.py

```diff
@@ -0,0 +1,295 @@
+"""
+Title: Text Image Denoiser.
```

Comment:
@anish9 The title can be a bit more elaborate. Some pointer

```
- Text Image Denoiser for Improving the OCR System
- Text Image Denoiser with a U-Net-like architecture 
- etc
```
You don't have to choose the exact word of the above example, but it may help you to relate.